app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,10 @@
+class TagsController < ApplicationController
+  def index
+    @tags = ActsAsTaggableOn::Tag.all
+  end
+
+  def show
+    @tag = ActsAsTaggableOn::Tag.find(params[:id])
+    @organizations = Organization.tagged_with(@tag.name)
+  end
```

Comment:
I believe so? #index is for getting all the tags for the sidebar list, and #show displays all the organizations for a particular tag.